docs/run-node/run-node.md

```diff
@@ -53,7 +53,7 @@ Now that your account has some tokens, you need to add a validator to your chain
 
 ```bash
 # Create a gentx.
-simd gentx my_validator --amount 100000stake --chain-id my-test-chain --keyring-backend test
+simd gentx --name my_validator --amount 100000stake --chain-id my-test-chain --keyring-backend test
```

Comment:
I get:

```
cosmos-sdk git:(master) ./build/simd gentx --name my_validator --amount 100000stake --chain-id my-test-chain --keyring-backend test
Error: unknown flag: --name
Usage:
  simd gentx [key_name] [flags]
```

Reply:
I am using version v0.39.1, and I know why this parameter doesn't work in this version. Thanks.

Changelog v0.40.0-rc0 - 2020-10-13:
```
(x/genutil) #6651 The gentx command has been improved. No longer are --from and --name flags required. Instead, a 
single argument, name, is required which refers to the key pair in the Keyring. In addition, an optional --moniker flag can 
be provided to override the moniker found in config.toml.
```